GVFS/GVFS.Platform.Mac/MacPlatform.cs

```diff
@@ -38,5 +39,28 @@ public override string GetDataRootForGVFSComponent(string componentName)
         {
             return new MacFileBasedLock(fileSystem, tracer, lockPath);
         }
+
+        public class MacPlatformConstants : POSIXPlatformConstants
+        {
+            public override string InstallerExtension
+            {
+                get { return ".dmg"; }
+            }
+
+            public override string GVFSBinDirectoryPath
+            {
+                get { return Path.Combine("/usr", "local", this.GVFSBinDirectoryName); }
+            }
+
+            public override string GVFSBinDirectoryName
+            {
+                get { return "vfsforgit"; }
+            }
+
+            public override string GVFSExecutableName
```

Comment:
There is no need to override `GVFSExecutableName`, its implementation is identical to the implementation in `POSIXPlatformConstants`.